lib/lf/geometry/point.cc

```diff
@@ -16,9 +16,8 @@ Eigen::MatrixXd Point::JacobianInverseGramian(
   return Eigen::MatrixXd::Zero(DimGlobal(), 0);
 }
 
-// NOLINTNEXTLINE(misc-unused-parameters)
 Eigen::VectorXd Point::IntegrationElement(const Eigen::MatrixXd& local) const {
-  return Eigen::Matrix<double, 1, 1>::Constant(1.0);
+  return Eigen::MatrixXd::Ones(local.cols(), 1);
```

Comment:
:+1:

Reply:
Well, I think it should be 
~~~
return Eigen::VectorXd::Ones(local.cols());
~~~
Otherwise a type conversion is necessary from a fully dynamic matrix to one with a fixed second dimension.

Reply:
yes I agree.